src/storage/exec/UpdateNode.h

```diff
@@ -53,7 +53,7 @@ class UpdateNode  : public RelNode<T> {
     kvstore::ResultCode getDefaultOrNullValue(const meta::SchemaProviderIf::Field* field,
                                               const std::string& name) {
         if (field->hasDefault()) {
-            props_[name] = field->defaultValue();
+            props_[field->name()] = Expression::eval(field->defaultValue(), *expCtx_);
```

Comment:
In update, `expCtx_ `   is `StorageExpressionContext`*

Is it okay to use it like this here?